ex1/ex1.c

```diff
@@ -25,14 +25,14 @@ int main(void)
 
         (*x)++;
         printf("%d Child proccess 3 %d\n", *x, pid);
-        // exit(9);
+        // exit(1);
```

Comment:
this number is pretty arbitrary. I imagine there's some conventions out there, but the biggest takeaway is: 0 means everything's fine, any other number means an error